### PR TITLE
feat: add lines to graphs and add legends

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard19**
## Summary

Currently, four charts are missing legends and/or need to remove the fill below the line.

- Added `forceLegend` prop to force a legend to be shown, even if it has one data point for the legend.
- Change area charts to line charts

### CODE REVIEW

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx [hunk 1]
```diff
@@ -185,13 +185,14 @@ export function SewerChart({ accessibility, dataAverages, dataPerInstallation, t
...
               {
-                type: 'area',
+                type: 'line',
                 metricProperty: 'average',
                 label: text.averagesDataLabel,
                 color: colors.scale.blue[3],
               },
             ]}
             dataOptions={dataOptions}
+            forceLegend
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:195]: It is impossible for me to add a comment to [line 164](https://github.com/minvws/nl-covid19-data-dashboard/pull/4542/files#diff-ba2f12c8cba10df698242039d52240b392f95f57e028880122096ec790cfd1e5R164), but is this also something that needs to be changed? Currently the sewer chart is still shown as an area graph when on VR or GM level and selection an installation.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 2]
```diff
@@ -1,9 +1,4 @@
-import {
-  colors,
-  NlTestedOverallValue,
-  TimeframeOption,
-  TimeframeOptionsList,
-} from '@corona-dashboard/common';
+import { colors, NlTestedOverallValue, TimeframeOption, TimeframeOptionsList } from '@corona-dashboard/common';
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:1]: Am I correct to assume that this file does not include any changes nor that it is in scope for this feature?

**Review by VWSCoronaDashboard26 — CHANGES_REQUESTED** [review]
Please have a look at my comment, as I think you might have missed something. Other than that, it looks good!

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 2 above)
Comment (VWSCoronaDashboard19) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:1]: Correct, it looks like I accidentally saved this file and caused a lot a formatting changes. This file can be ignored.

**Review by VWSCoronaDashboard26 — APPROVED** [review]

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx (hunk 1 above)
Comment (VWSCoronaDashboard19) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:195]: Thanks for the heads up, didn't check that one before. I changed it in my last commit.

**Review by VWSCoronaDashboard25 — APPROVED** [review]

**Pull request merged**